src/Common/src/Interop/Windows/kernel32/Interop.Heap.cs

```diff
@@ -3,15 +3,12 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Text;
-using System.Diagnostics;
 using System.Runtime.InteropServices;
-
 using Microsoft.Win32.SafeHandles;
 
-internal static partial class Interop
+internal partial class Interop
 {
-    internal static partial class Heap
+    internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
SetLastError on methods in this file is unnecessary overhead - none of the callers check the last error.